build.sbt

```diff
@@ -252,3 +230,173 @@ fixCSS := {
   if (("npm run fix-dark" #&& "npm run fix-light" !) != 0)
     throw new Exception("Error in CSS fix")
 }
+
+val pushCond                 = "github.event_name == 'push'"
+val prCond                   = "github.event_name == 'pull_request'"
+val masterCond               = "github.ref == 'refs/heads/master'"
+val geminiRepoCond           = "startsWith(github.repository, 'gemini')"
+val notDependabotCond        = "github.actor != 'dependabot[bot]'"
+def allConds(conds: String*) = conds.mkString("(", " && ", ")")
+def anyConds(conds: String*) = conds.mkString("(", " || ", ")")
+
+lazy val setupNode = WorkflowStep.Use(
+  UseRef.Public("actions", "setup-node", "v2"),
+  name = Some("Use Node.js"),
+  params = Map("node-version" -> "14", "cache" -> "npm"),
+  env = Map("FONTAWESOME_NPM_AUTH_TOKEN" -> "${{ secrets.FONTAWESOME_NPM_AUTH_TOKEN }}")
+)
+
+lazy val npmCache = WorkflowStep.Use(
+  UseRef.Public("c-hive", "gha-npm-cache", "v1"),
+  env = Map("FONTAWESOME_NPM_AUTH_TOKEN" -> "${{ secrets.FONTAWESOME_NPM_AUTH_TOKEN }}")
+)
+
+lazy val sbtStage = WorkflowStep.Sbt(List("stage"), name = Some("Stage"))
+
+// https://stackoverflow.com/a/55610612
+lazy val npmInstall = WorkflowStep.Run(
+  List(
+    "rm .npmrc",
```

Comment:
Could you elaborate what is tIs doing. Is it adding the fontawesome npm repo locally?

Reply:
I have no idea what this is doing :) I copied it from your existing workflow.

https://github.com/gemini-hlsw/explore/blob/a366ec0a2be4df7a844c45ffb8da1313f258c489/.github/workflows/ci.yml#L28-L34

Reply:
Right, this adds the fontawesome registry so we can use the paid version of their icons repo